sdk/eventhub/eventhubs-checkpoint-blob/src/log.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import debugModule from "debug";
+/**
+ * @ignore
+ * log statements for error
+ */
+export const error = debugModule("azure:event-hubs:error");
+/**
+ * @ignore
+ * log statements for blobPartitionManager
+ */
+export const blobPartitionManager = debugModule("azure:eventhubs-checkpoint-blob:blobPartitionManager");
```

Comment:
if you make this `azure:eventhubs-checkpoint-blob:info` it aligns with the logging guidelines and the work happening in #4926.

Reply:
Updated!